### PR TITLE
PROD-5544 Getsafe Base Rate Docs Update

### PR DESCRIPTION
**Pull request by reinisk22**
https://kaskoltd.atlassian.net/browse/PROD-5544

### CODE REVIEW

File: source/products/roland/getsafebaserate/rest-api.rst [hunk 1]
```diff
@@ -176,11 +176,11 @@ Example Request
 
     curl https://api.kasko.io/payments \
         -X POST \
-        -u <YOUR SECRET API KEY>: \
+        -u YOUR SECRET API KEY: \
```
Comment (Kemito) [inline source/products/roland/getsafebaserate/rest-api.rst]: Why removed <> ?

File: source/products/roland/getsafebaserate/rest-api.rst [hunk 2]
```diff
@@ -102,9 +102,9 @@ This request stores policy holder information that is related to offer. Followin
    "previous_insurance_cancellation", "integer|optional",  "Previous cancellation reason.",   "2"
    "previous_insurance_start_date",   "string|optional",   "Previous insurance start date in ISO 8601 format.",   "YYYY-mm-dd"
    "previous_insurance_end_date",     "string|optional",   "Previous insurance start date in ISO 8601 format.",   "YYYY-mm-dd"
-   "partner_coverage",                "bool",              "Partner coverage.",   "true"
-   "coinsured_first_name",            "string|optional",   "Co-insured First name. Required if ``partner_coverage`` is ``true``.",   "FirstName"
-   "coinsured_last_name",             "string|optional",   "Co-insured Last name. Required if ``partner_coverage`` is ``true``.",   "LastName"
+   "partner_coverage_type",           "string",            "Partner coverage type. in:single,spouse,partner",     "single"
+   "coinsured_first_name",            "string|optional",   "Co-insured First name. Required if ``partner_coverage_type`` is ``partner``.",   "FirstName"
```
Comment (Kemito) [inline source/products/roland/getsafebaserate/rest-api.rst]: "First" => "first" & "Last" => "last" ?

Comment (reinisk22) [reply source/products/roland/getsafebaserate/rest-api.rst]: Updated

File: source/products/roland/getsafebaserate/rest-api.rst (hunk 1 above)
Comment (reinisk22) [reply source/products/roland/getsafebaserate/rest-api.rst]: Quality of life feature to make it easier to copy/paste the example request ids. So far our docs don't have a consistent rule for this, therefore I chose this style.

**Pull request merged**